src/components/TaskItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+interface ItemName {
+  name: string;
+  key: string;
+}
+
+export default function TaskItem({ name, key }: ItemName) {
+  return (
+    <li className="todo-stack-small">
+      <div className="c-cb">
+        <input id={key} type="checkbox" />
```

Comment:
esse ID é realmente necessário? Se não for, recomendo só removê-lo. não tem um motivo prático no react para usar `id`. Isso vai ser mais usado quando formos falar de testes automatizados.